lib/yang.h

```diff
@@ -34,7 +34,7 @@ extern "C" {
 #endif
 
 /* Maximum XPath length. */
-#define XPATH_MAXLEN 512
+#define XPATH_MAXLEN 1024
```

Comment:
why is this necessary? Are we just in a race of just increasing this until it just works?  I specifically asked was 512 enough not more than a month or so ago.  What's changed?

Reply:
@donaldsharp we brought this up for discussion on yang and also in NB meeting. For staticd max xpath len is 585 and we anticipate that in BGP it may be more than that. As per Yang channel discussion we will increase when we have a need.